static/app/views/issueDetails/header.tsx

```diff
@@ -139,6 +139,9 @@ function GroupHeaderTabs({
       >
         {t('Merged Issues')}
       </TabList.Item>
+      <TabList.Item key={Tab.RELATED_ISSUES} to={`${baseUrl}related/${location.search}`}>
```

Comment:
Do you want to hide this with a feature flag? If so you'll want to use the `hidden` prop.